pandas/tests/indexing/test_partial.py

```diff
@@ -637,3 +637,66 @@ def test_partial_set_empty_frame_empty_consistencies(self):
         df.loc[0, 'x'] = 1
         expected = DataFrame(dict(x=[1], y=[np.nan]))
         tm.assert_frame_equal(df, expected, check_dtype=False)
+
+    def test_access_timezoned_datetimeindex_with_timezoned_label_utc(self):
+
+        # GH 6785
```

Comment:
I think this is an incorrect issue number

Reply:
oh let me have a look how that could happen